LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 [Mátyás Gombos],
```

Comment:
- [ OPTIONAL ] Great job working on the README file but here I think you have the wrong year since we are in 2023 aren't we? 😉😊💯 so kindly change it to 2023 instead of 2022

Reply:
@Reem-lab good catch! 😆 i will fix it!